### PR TITLE
fixed missing error if a build property is provided

### PR DESCRIPTION
**Pull request by malud**
Fix for https://github.com/sloppyio/sloppose/issues/5

### CODE REVIEW

**Comment by jzaefferer** [comment]
Tested this with `./sloppose convert pkg/converter/testdata/docker-compose-v2-reject.yml`

That failed with these lines (plus usage):

```
ERRO[0000] Failed to parse service busy_env: open /Users/jzaefferer/dev/sloppy/sloppose/testdata/test.env: no such file or directory
ERRO[0000] Could not parse config for project sloppy : open /Users/jzaefferer/dev/sloppy/sloppose/testdata/test.env: no such file or directory
Error: open /Users/jzaefferer/dev/sloppy/sloppose/testdata/test.env: no such file or directory
```
The two `ERRO[0000]` lines look out of place.

Removing the test.env reference, I got this instead:
```
(0x1641c20,0xc420218e00)
Error: the build property is not supported, please specify an image instead
```
That first line shouldn't be there, right?

File: README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ Checkout to `$GOPATH/src/github.com/sloppyio/sloppose`
 
 Create a development build within the Go environment: `make build-dev`
 
-Create an osx build with docker: `docker run -v $PWD:/go/src/github.com/sloppyio/sloppose --workdir /go/src/github.com/sloppyio/sloppose -e GOOS=darwin golang:1.8.3 make build-dev`
+Create an osx build with docker: `docker run -v $PWD:/go/src/github.com/sloppyio/sloppose --workdir /go/src/github.com/sloppyio/sloppose -e GOOS=darwin golang:1.9.4 make build-dev`
```
Comment (jzaefferer) [inline README.md]: How about a `make dev` (or `make beta`) as in the CLI repo that includes the Docker run command?

File: pkg/converter/sloppy_file_test.go [hunk 2]
```diff
@@ -89,3 +93,13 @@ func TestNewSloppyFileInvalidPorts(t *testing.T) {
 		}
 	}
 }
+
+func TestNewComposeFileBuildProperty(t *testing.T) {
+	helper := test.NewHelper(t)
+	cf, err := loadComposeFile("testdata/docker-compose-v2-reject.yml", "no-build-supported")
+	helper.Must(err)
+	_, err = converter.NewSloppyFile(cf)
+	if err == nil {
```
Comment (jzaefferer) [inline pkg/converter/sloppy_file_test.go:102]: This looks like it should also test that the error, when present, has the right message.

**Comment by malud** [comment]
Those `ERRO[0000]` errors are fine there. Since the current working dir and compose file context ist wrong in your test example.

File: pkg/converter/sloppy_file_test.go (hunk 2 above)
Comment (malud) [reply pkg/converter/sloppy_file_test.go:102]: Could :), I will add the err string comparison

**Comment by jzaefferer** [comment]
> Since the current working dir and compose file context ist wrong in your test example.

Alright, with the right workdir I'm still getting the 2nd error:

```
sloppose/pkg/converter/testdata $ ../../../sloppose convert docker-compose-v2-reject.yml
(0x1676c40,0xc42018cde0)
Error: the build property is not supported, please specify an image instead
```

PS: Incremental reviews are much easier when you push changes as separate commits, instead of amending.

**Comment by jzaefferer** [comment]
Looks good!

**Pull request merged**